oe_webtools_analytics/oe_webtools_analytics.behat.inc

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/**
+ * @file
+ * Contains \OEParagraphsSubContext.
+ */
+
+declare(strict_types = 1);
+
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Drupal\DrupalExtension\Context\DrupalSubContextBase;
+use Drupal\oe_webtools_analytics\Utils\WebtoolsAnalyticsIndex;
+
+/**
+ * Behat step definitions for testing the OE Webtools Analytics module.
+ */
+class OEWebtoolsAnalyticsSubContext extends DrupalSubContextBase {
+
+  /**
+   * Setup demo site.
+   *
+   * @BeforeScenario @webtoolsAnalytics
+   */
+  public function setupWebtoolsAnalytics(BeforeScenarioScope $scope) {
+    \Drupal::service('module_installer')->install(['oe_webtools_analytics']);
+  }
+
+  /**
+   * Checking if the settings on the page match the value on config.
+   *
+   * @Then the rendered script should contains the :arg1 with the same value as the configuration set in settings file
```

Comment:
Better pass in the expected value as an argument, so that we can make real assertions.
In a test it is better to be as explicit (hard-coded) as possible.
I would propose a signature like
Then the analytics script should have (the ):property set to :value
(where :property would be SiteID and :value 12345)